config/src/config/network_config.rs

```diff
@@ -343,6 +344,46 @@ impl NetworkConfig {
         }
         Ok(())
     }
+
+    /// Retrieve and merge seeds so that they have all keys associated
```

Comment:
Seems like a simple file move/refactor, right? This could be a separate PR 😄

Reply:
https://github.com/aptos-labs/aptos-core/pull/12638